go.mod

```diff
@@ -58,7 +58,7 @@ require (
 	github.com/btcsuite/btcd v0.22.0-beta // indirect
 	github.com/cespare/xxhash v1.1.0 // indirect
 	github.com/cespare/xxhash/v2 v2.1.2 // indirect
-	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
+	github.com/cpuguy83/go-md2man/v2 v2.0.1 // indirect
```

Comment:
Should these go.mod dependency upgrades be discarded? Aren't they out of scope?

Reply:
>  Aren't they out of scope?

May be.

But, all of those go.mod and go.sum changes are done by `go mod tidy`. As long as it is not hurting the code, I see no reason to go in it and manually change things!

Reply:
That's strange, `go mod tidy` doesn't do that for me. Please re-check?

Reply:
reverted go.mod changes and ran `go mod tidy` again!

Reply:
```sh
git checkout kishan/fix/dont-ban-us
git reset d2ec68d3e319d08f32dc0df4adf1fd7531d12217
git checkout go.mod go.sum
go mod tidy
```

does not produce any change, make sure you revert go.sum as well before re-running `go mod tidy`?